tests/pos-special/repeatedArgs213.scala

```diff
@@ -1,38 +0,0 @@
-import scala.collection.{immutable, mutable}
-import java.nio.file.Paths
-
-// Missing from 2.12 standard library
-package scala.runtime {
-  object ScalaRunTime {
-    abstract class ArraySeq[+A] extends immutable.Seq[A]
-
-    def genericWrapArray[T](xs: Array[T]): ArraySeq[T]          = ???
-    def wrapRefArray[T <: AnyRef](xs: Array[T]): ArraySeq[T]    = ???
-    def wrapIntArray(xs: Array[Int]): ArraySeq[Int]             = ???
-    def wrapDoubleArray(xs: Array[Double]): ArraySeq[Double]    = ???
-    def wrapLongArray(xs: Array[Long]): ArraySeq[Long]          = ???
-    def wrapFloatArray(xs: Array[Float]): ArraySeq[Float]       = ???
-    def wrapCharArray(xs: Array[Char]): ArraySeq[Char]          = ???
-    def wrapByteArray(xs: Array[Byte]): ArraySeq[Byte]          = ???
-    def wrapShortArray(xs: Array[Short]): ArraySeq[Short]       = ???
-    def wrapBooleanArray(xs: Array[Boolean]): ArraySeq[Boolean] = ???
-    def wrapUnitArray(xs: Array[Unit]): ArraySeq[Unit]          = ???
-  }
-}
```

Comment:
The definitions above should be removed but apart from that we should keep this test and move it to `tests/pos`

Reply:
Indeed, updated!